diff --git a/scripts/train.py b/scripts/train.py
index e4b7af9..5ca44aa 100644
--- a/scripts/train.py
+++ b/scripts/train.py
@@ -260,3 +260,10 @@ if __name__ == "__main__":
 
         if args.first_variant:
             exit()
+
+
+# To do:
+# Understand what the videos in data mean
+# Just change debug mode parameters: Run wipe for 1 epoch and see output (maybe even force it to execute reach for specific (x,y,z))
+# Change controller and see what parameters are needed
+# Update parameters based on 1 epoch output so that it now completely works with impedance controller and uses impedance parameters
\ No newline at end of file
