diff --git a/scripts/train.py b/scripts/train.py
index e4b7af9..1ac7911 100644
--- a/scripts/train.py
+++ b/scripts/train.py
@@ -121,6 +121,7 @@ base_variant = dict(
     ),
 )
 
+
 env_params = dict(
     lift={
         'env_variant.env_type': ['Lift'],
@@ -201,7 +202,7 @@ def process_variant(variant):
         variant['algorithm_kwargs']['min_num_steps_before_training'] = steps
         variant['algorithm_kwargs']['num_trains_per_train_loop'] = 50
         variant['replay_buffer_size'] = int(1E3)
-        variant['dump_video_kwargs']['columns'] = 2
+        variant['dump_video_kwargs']['columns'] = 3
 
     if args.no_video:
         variant['save_video'] = False
@@ -255,8 +256,14 @@ if __name__ == "__main__":
             use_gpu=(not args.no_gpu),
             gpu_id=args.gpu_id,
             mode='local',
-            num_exps_per_instance=1,
         )
 
         if args.first_variant:
             exit()
+
+
+# To do:
+# Understand what the videos in data mean
+# Just change debug mode parameters: Run wipe for 1 epoch and see output (maybe even force it to execute reach for specific (x,y,z))
+# Change controller and see what parameters are needed
+# Update parameters based on 1 epoch output so that it now completely works with impedance controller and uses impedance parameters
\ No newline at end of file
